app.py

```diff
@@ -23,7 +23,10 @@
 
 # Widgets
 initial_infections = st.sidebar.number_input(
-    "Currently Known Regional Infections", value=known_infections, step=10, format="%i"
+    "Currently Known Regional Infections", min_value=1, value=known_infections, step=10, format="%i"
+)
+current_hosp = st.sidebar.number_input(
```

Comment:
@fbenamy or @themightychris do you know why this needs to be added? The currently live version shows this value... how is it not in the code already? 

![image](https://user-images.githubusercontent.com/47401552/76923591-10d28480-6891-11ea-9c29-d220624b3452.png)

Looking deeper, but this is my only concern with an otherwise 👍 PR

Reply:
Well, the obvious interpretation of what I'm seeing seems to be true. A previous change must have removed Currently Hospitalized COVID-19 Patients. 

I don't want to add an input that doesn't get used downstream, so let's check and see where it is used...